EasyReflectometryApp/Logic/Proxies/Simulation.py

```diff
@@ -189,6 +189,8 @@ def _updateCalculatedData(self):
         sld_profile = self.parent._interface.sld_profile(to_use.uid)
 
         self.parent._plotting_1d_proxy.setCalculatedData(x, y)
+        self.parent._plotting_1d_proxy.setBackgroundData(x, np.ones_like(x) * to_use.background.raw_value)
+        self.parent._plotting_1d_proxy.setScaleData(x, np.ones_like(x) * to_use.scale.raw_value)
```

Comment:
minor nitpick but 
```
np.repeat( to_use.background.raw_value, (x))
```
is quicker and maybe even more readable
https://stackoverflow.com/questions/5891410/numpy-array-initialization-fill-with-identical-values

Reply:
Woah, I have never heard of `np.repeat` 😱

Reply:
I changed to the fastest option of `a = np.empty(x.size); a.fill(b)`.